CharacterClient/main.py

```
@@ -0,0 +1,66 @@
+def main():
+    parser = argparse.ArgumentParser(description="DreamWeaver Character Client")
+    parser.add_argument(
+        "--server_url",
+        type=str,
+        default=os.getenv("SERVER_URL", "http://127.0.0.1:8000"),
+        help="URL of the DreamWeaver Server (default: env SERVER_URL or http://127.0.0.1:8000)"
+    )
+    parser.add_argument(
+        "--Actor_id",
+        type=str,
+        default=os.getenv("CLIENT_Actor_ID", "Actor_Client"),
```

Ruff (0.11.9)

Use capitalized environment variable `CLIENT_ACTOR_ID` instead of `CLIENT_Actor_ID`

(SIM112)